app/controllers/people_controller.rb

```diff
@@ -16,11 +16,20 @@ def create
     end
   end
 
+  def search_by_email
+    result = Person.where("email = '#{params['email']}'").limit(params['per_page']).offset((params['page'] - 1) * params['per_page'])
+    render :json => result.to_json
+  end
+
+  def search_by_name
+    result = Person.where("UPPER(name) like '%#{params['name'].upcase}%'").limit(params['per_page']).offset((params['page'] - 1) * params['per_page'])
```

Comment:
Same concerns here with sql injection as above. How could we change this query to make it safe from sql injection?

Reply:
@jdguzman 
I fixed this one.